Tests/eZ/Publish/Slot/AbstractSlotTest.php

```diff
@@ -6,9 +6,12 @@
 namespace EzSystems\RecommendationBundle\Tests\eZ\Publish\Slot;
 
 use PHPUnit\Framework\TestCase;
+use eZ\Publish\Core\Base\Tests\PHPUnit5CompatTrait;
 
 abstract class AbstractSlotTest extends TestCase
 {
+    use PHPUnit5CompatTrait;
```

Comment:
I think this one can be removed, as in:
https://github.com/ezsystems/EzSystemsRecommendationBundle/pull/118
https://github.com/ezsystems/ezpublish-kernel/pull/2446